discovery-provider/src/queries/get_attestation.py

```diff
@@ -57,7 +57,7 @@ def _get_combined_id(self):
         return f"{self.challenge_id}:{self.challenge_specifier}"
 
     def _get_encoded_amount(self):
-        amt = int(self.amount) * 10 ** WAUDIO_DECIMALS
+        amt = int(self.amount) * 10**WAUDIO_DECIMALS
```

Comment:
i know this is probably some lint thing but agreed that we should scope these types of changes separately in a different PR